committer/src/setup.rs

```diff
@@ -70,6 +70,39 @@ pub fn block_committer(
     )
 }
 
+pub fn state_committer(
+    l1: L1,
+    storage: impl Storage + 'static,
+    _registry: &Registry,
+    cancel_token: CancellationToken,
+    config: &config::Config,
+) -> tokio::task::JoinHandle<()> {
+    let state_committer = services::StateCommitter::new(l1, storage);
+
+    schedule_polling(
+        config.app.block_check_interval,
+        state_committer,
+        "State Committer",
+        cancel_token,
+    )
+}
+
+pub fn state_importer(
+    fuel: FuelApi,
+    storage: impl Storage + 'static,
+    _registry: &Registry,
+    cancel_token: CancellationToken,
+) -> tokio::task::JoinHandle<()> {
+    let state_importer = services::StateImporter::new(storage, fuel);
+
+    schedule_polling(
+        Duration::from_secs(10),
```

Comment:
we should either add a new config for this or use `config.app_block_check_interval` ?